dependency-manager-tests/spm/Package_swift.template

```diff
@@ -0,0 +1,22 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SPMProject",
+    platforms: [
+        .macOS(.v10_14),
```

Comment:
could it make sense to test dependencies on both supported platforms?

Reply:
I just checked and seems that it will be ignored:
* adding `.iOS(.v11)` (so targeting under SDK supported target) doesn't break this test;
* changing to `.macOS(.v10_13)` (again, under SDK's supported target) does break it (`error: the product 'Datadog' requires minimum platform version 10.14 for macos platform`).

This is because this test is configured to build and run basic macOS CLI app, so the iOS platform dependency is ignored and not checked. This is enough to validate if the SDK's `Package.swift` on current branch is consistent for at least one platform.

To make it better, another test might be added, this time using iOS app `.xcodeproj` scaffold configured to use SDK's `Package.swift` for current branch. I will create separate JIRA ticket to improve this in the future - not a prio right now.